### PR TITLE
Cms 1633712910222/playbooks/how to join a dao community

### PR DESCRIPTION
**Pull request by Techniciann9ne**
Correction

### CODE REVIEW

File: docs/playbooks/how-to-join-a-dao-community.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: 📚 How to Join DAO Community
 by @Techniciann9ne
 
 
-- How non developers can make an impact in the dao space
+- How non-developers can make an impact in the DAO space
```
Comment (luxumbra) [inline docs/playbooks/how-to-join-a-dao-community.md:8]: There's a conflict in this file (i think caused by the earlier heading change i suggested) which is preventing it from doing a clean merge. I don't think you can fix it in the CMS, so will need to be done manually. I can help you do that if you like but will have to wait till after the weekend. 👍🏻

**Review by luxumbra — CHANGES_REQUESTED** [review]
We need to resolve the conflicts before we can merge. :)

File: docs/playbooks/how-to-join-a-dao-community.md (hunk 1 above)
Comment (Baconaetor) [reply docs/playbooks/how-to-join-a-dao-community.md:8]: Huh yeah that's weird. @luxumbra did you close out the other PR already as I only see this one. I'll see if I can handle the merge conflicts after I review.

**Review by Baconaetor — CHANGES_REQUESTED** [review]
The section you have with the slashes, can you break that out into a bulleted list? That will make it cleaner. 

Also, while you're in here, can you add the "A" that's missing into the title?

**Review by Baconaetor — APPROVED** [review]
Gonna go a head and approve and merge this. These are good changes and we resolved the conflict. I do see some typos/errors in the wording of the page, but to prevent this from getting messier, we can save that for another PR.

**Pull request merged**